lib/appTemplates.js

```diff
@@ -76,7 +76,7 @@ function getTemplates(config) {
       "type": "template-partial",
       "source": `
   <div class="checkbox-favorites">
-    <input type="checkbox" value="0" id="{{state "pageInitId"}}"/>
+    <input type="checkbox" value="0" id="{{state "pageInitId"}}" aria-label="page bookmark"/>
```

Comment:
At some point, the label will need localization, but I don't think we are at that point